local_costmap:
  global_frame: odom
  robot_base_frame: base_link
  update_frequency: 5.0
  publish_frequency: 2.0
  static_map: false

  rolling_window: true
  width: 4.0
  height: 4.0
  resolution: 0.05

  plugins:
  - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
  - {name: inflation_layer, type: "costmap_2d::InflationLayer"}
  obstacle_layer:
    observation_sources: laser_scan_sensor
    laser_scan_sensor: {sensor_frame: base_laser_link, data_type: LaserScan, topic: /scan_raw, marking: true, clearing: true}
  inflation_layer:
    inflation_radius: 0.1
    cost_scaling_factor: 10.0